internal/resources/storage_statefulset.go

```diff
@@ -180,7 +241,13 @@ func (b *StorageStatefulSetBuilder) buildContainer() corev1.Container { // todo
 		},
 
 		SecurityContext: &corev1.SecurityContext{
-			Privileged: ptr.Bool(false),
+			Capabilities: &corev1.Capabilities{
+				Add: []corev1.Capability{
+					"SYS_RAWIO",
+				},
+			},
+			Privileged: ptr.Bool(true),
```

Comment:
This is just temporarily for testing. Symlinks are owned by root\root, I need to figure out how to re-own them as root\disk in all environments, then this privilege escalation won't be necessary.